tests/data/lorem_ipsum.ris

```diff
@@ -0,0 +1,49 @@
+1.
+TY  - JOUR
+ID  - 12345
+T1  - Title of reference
+A1  - Marx, Karl
+A1  - Lindgren, Astrid
+A2  - Glattauer, Daniel
+Y1  - 2014//
```

Comment:
`Y1` seems to correspond to the `year` field and `N1` to the `abstract`. `IS` should be `number` and `SN` the `pages`.